eng/ignore-links.txt

```diff
@@ -15,7 +15,12 @@ https://docs.microsoft.com/javascript/api/@azure/arm-communication
 https://docs.microsoft.com/javascript/api/@azure/arm-databoxedge
 https://docs.microsoft.com/javascript/api/@azure/arm-datadog
 https://docs.microsoft.com/javascript/api/@azure/arm-healthbot
+https://docs.microsoft.com/javascript/api/@azure/arm-frontdoor
+https://docs.microsoft.com/javascript/api/@azure/arm-domainservices
 https://docs.microsoft.com/javascript/api/@azure/arm-deploymentmanager?view=azure-node-preview
 https://docs.microsoft.com/javascript/api/@azure/arm-hanaonazure?view=azure-node-preview
 https://docs.microsoft.com/javascript/api/@azure/arm-kubernetesconfiguration?view=azure-node-preview
 https://docs.microsoft.com/en-us/azure/azure-resource-manager/deployment-manager-overview
+https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/deviceupdates/iot-device-update-rest
+https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/deviceupdate/iot-device-update-rest/samples/v1/javascript/devices.js
+https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/deviceupdate/iot-device-update-rest/samples/v1/typescript/src/devices.ts
```

Comment:
You shouldn't need to add these links here. CI should resolve these within the branch.

/cc @praveenkuttappan

Reply:
No, as the codes has not merge, they return 404. So when onboarding new sdk, we add them to ignore links, and delete them later.

Reply:
@praveenkuttappan we discussed this offline, CI should be able to handle this. Do you see anything missing that is causing CI to tag these as broken links?

Reply:
yes, in releasing mgmt sdk, we encounter it frequently.